### ISSUE TITLE
Empty values are inserted as null strings

### ISSUE THREAD

**Issue by fazendeiro** [sqlite-sync/SQLite-sync.com#34]
Hello,

I'm developing an Android app synced with a MSSQL Server.
After adding tables and syncing, i'm encountering an issue because there is no null values in my SQLite database, all has been inserted as null strings. It happens with TEXT columns, but also INT, that causes conversion errors in my app.

Is there a setting to change that?

**Comment by IchoTolot** [comment]
As far as I know changeable settings are only in the .conf file

**Comment by fazendeiro** [comment]
Thank you, 
I haven't found anything that changes that. Do you think this is normal behavior?

**Comment by IchoTolot** [comment]
I don't think so. IMHO all database values will be converted to string because they are transmitted as xml. But they should be converted back before they are written into the database. Is the Sqlite datatype of the column really "text"?
Personally I would test this with a mysql database, because in my opinion the support is best here. I run in some trouble too when I use postgres.
And...most important. Always use the last version from https://projects.ampliapps.com/projects/ampli-sync/files

**Comment by fazendeiro** [comment]
I use 3.2.15 so I don't think that's the issue.
Yes it is realy text, or varchar columns. The problem is that happens on integer columns.
I changed from Room library to SQLiteOpenHelper wich is more configurable.

Thank you for answers !
